contentheader/package.json

```
@@ -0,0 +1,82 @@
+{
+  "name": "contentheader",
+  "version": "0.1.0",
+  "description": "Minimal JupyterLab example using contentHeader.",
+  "keywords": [
+    "jupyter",
+    "jupyterlab",
+    "jupyterlab-extension"
+  ],
+  "homepage": "https://github.com/jupyterlab/extension-examples",
+  "bugs": {
+    "url": "https://github.com/jupyterlab/extension-examples/issues"
+  },
+  "license": "BSD-3-Clause",
+  "author": {
+    "name": "Project Jupyter Contributors",
+    "email": ""
+  },
+  "files": [
```

Don't hesitate to add yourself here:

```suggestion
  "contributors": [
    { "name": "Ahmed Fasih" }
  ],
  "files": [
```